drivers/counter/counter_nrfx_rtc.c

```diff
@@ -222,9 +283,14 @@ static int init_rtc(struct device *dev,
 		return -EBUSY;
 	}
 
-	get_dev_data(dev)->top = COUNTER_MAX_TOP_VALUE;
+	err = ppi_setup(dev);
+	if (err != 0) {
+		return err;
+	}
+
+	data->top = COUNTER_MAX_TOP_VALUE;
 
-	LOG_INST_DBG(nrfx_config->log, "Initialized");
+	LOG_INST_DBG(nrfx_config->log, "Initialized %p", dev);
```

Comment:
Wouldn't `%s", dev->config->name` be more convenient?

Reply:
`%p` was a leftover from debuging. Since `LOG_INST_..` is used there is no need to add additional instance description.